src/liballoc/boxed.rs

```diff
@@ -239,6 +239,17 @@ impl<T> Box<T> {
     pub fn pin(x: T) -> Pin<Box<T>> {
         (box x).into()
     }
+
+    /// Converts a `Box<T>` into a `Box<[T]>`
+    ///
+    /// This conversion does not allocate on the heap and happens in place.
+    ///
+    /// This is also available via [`From`].
```

Comment:
Is this true?

Reply:
Ops, thanks. I just removed the From impl because it's insta-stable